Controllers/tutorials.js

```diff
@@ -1,8 +1,9 @@
-const Tutorial = require("../Models/tutorials");
+import Tutorial from '../Models/tutorials.js';
 
 const getAllTutorials = async (req, res) => {
+  const { userId } = req;
   try {
-    const allTutorials = await Tutorial.find({});
+    const allTutorials = await Tutorial.find({ _id: userId });
```

Comment:
I believe you should search for the field userId here, not for the _id.

Reply:
Yep, my mistake, thank you.